integrations/manual/swaggerui/README.md

```diff
@@ -0,0 +1,4 @@
+This will serve a swagger-ui webinterface for trying out & viewing your datafire endpoints.
+
+> NOTE: executing endpoints only works on productionservers (datafire on port 80). This is because datafire does not specify the port in the baseurl of /jsonapi.json.
```

Comment:
Note: you can specify top-level fields for your `openapi.json` in `DataFire.yml`.  E.g.

```yaml
openapi:
  host: api.example.com:3000
  schemes:
    - https
```

Also, I think `/jsonapi.json` should be `/openapi.json`